packages/aws-cdk-lib/aws-secretsmanager/lib/rotation-schedule.ts

```diff
@@ -2,6 +2,7 @@ import { Construct } from 'constructs';
 import { ISecret, Secret } from './secret';
 import { CfnRotationSchedule } from './secretsmanager.generated';
 import * as ec2 from '../../aws-ec2';
+import { Schedule } from '../../aws-events';
```

Comment:
AHhhh I suppose this is ok because the schedule is not exposed in anyway, but we STILL need to get this into class into core.